### PR TITLE
PUD-1281 - Delete address

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: cypress/integration/stepDefinitions/book.js [hunk 1]
```diff
@@ -82,6 +82,18 @@ When('Maria can see the addresses listed', () => {
     cy.clickButton('Continue')
 })
 
+When('Maria deletes one of the last known addresses', () => {
+    cy.clickLink("Change address 1")
+    cy.pageHeading().should('equal', 'Last known addresses')
+    cy.clickButton("Delete address 1")
+    const address = recall.lastKnownAddresses[0]
```
Comment (philsegal) [inline cypress/integration/stepDefinitions/book.js:89]: will this have refreshed the recall list? Or is this the original list?

File: cypress/integration/stepDefinitions/book.js [hunk 2]
```diff
@@ -82,6 +82,18 @@ When('Maria can see the addresses listed', () => {
...
+    cy.recallInfo('Address').should('contain', address.line1)
+    cy.selectRadio('Do you want to add another address?', 'No')
+    cy.clickButton('Continue')
+    cy.pageHeading().should('equal', 'Check the details before booking this recall')
+    cy.recallInfo('Address').should('contain', address.line1)
```
Comment (philsegal) [inline cypress/integration/stepDefinitions/book.js:94]: I *think* this is asserting that we only have 1 address now, but its not clear. Is there a way to assert that there is just 1 address?

File: cypress/integration/stepDefinitions/book.js (hunk 1 above)
Comment (jonnywyatt) [reply cypress/integration/stepDefinitions/book.js:89]: recall list isn't involved in this test, do you mean recall info?

**Review by philsegal — APPROVED** [review]
Thankyou 🙏

**Pull request merged**